src/agent/onefuzz-supervisor/src/main.rs

```diff
@@ -145,7 +145,7 @@ async fn run_agent(config: StaticConfig) -> Result<()> {
         Box::new(worker::WorkerRunner),
     );
 
-    info!("running supervisor agent");
+    info!("running agent");
```

Comment:
Why be less specific?

Reply:
This is more correct.  it's launching onefuzz-agent

Reply:
@bmc-msft, FWIW, it really is invoking the _supervisor_ agent. The worker agent (`onefuzz-agent`) is invoked in [`WorkerRunner::run()`](https://github.com/bmc-msft/onefuzz/blob/a998f860dca4cb21740b60a56eb094bfb939b3da/src/agent/onefuzz-supervisor/src/worker.rs#L216).

Also, there is now a "supervisor agent finished" log message a few lines below that no longer matches this one.